### ISSUE TITLE
Minimum Monitors for Class Pool does not support "all"

### ISSUE THREAD

**Issue by amsinha2** [F5Networks/f5-appsvcs-extension#106]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```
3.11.0
```

##### BIGIP VERSION

```
13.1.0.2
```

##### SUMMARY

```
When declaring a pool using AS3 the default behavior for minimum monitors is set to "at least 1".  
The default behavior when creating a new pool is "Availibility requirement" is set to all. 

AS3 does not support "all"

```

##### STEPS TO REPRODUCE

```
"pool1": {
                    "class": "Pool",
                    "loadBalancingMode": "least-connections-member",
                    "minimumMonitors": "all",
                    "monitors": [
                        {
                            "bigip": "/Common/Monitor1"
                        },
{
                            "bigip": "/Common/Monitor2"
                        }
                    ],
                    "members": [
                        {
                            "servicePort": 8080,
                            "addressDiscovery": "fqdn",
                            "hostname": "poolserver1.qad.root"
                        }
                    ]
                },
```

##### EXPECTED RESULTS

```
ltm pool PartitionA/Shared/pool1 {
    monitor Common/Monitor1 and Common/Monitor2 
}
```

##### ACTUAL RESULTS

```
{
    "code": 422,
    "errors": [
        PartitionA/Shared/pool1/minimumMonitors: should be integer"
    ],
    "declarationFullId": "",
    "message": "declaration is invalid"
}

ltm pool PartitionA/Shared/pool1 {
    monitor min 2 of { Common/Monitor1 and Common/Monitor2 }
}

```

**Comment by focrensh** [comment]
Thanks for the feedback. It looks like 0 also does not work. AT-436 opened for internal tracking.

**Comment by zivvel** [comment]
"all" works for me in 3.13.0

**Comment by dstokesf5** [comment]
It looks like this was resolved in 3.13.0, closing this issue. If you are still experiencing a problem please reopen this issue or create a new one.
